pkg/deploy/api/v1/conversion.go

```diff
@@ -143,24 +206,6 @@ func convert_v1_DeploymentTriggerImageChangeParams_To_api_DeploymentTriggerImage
 	if err := s.Convert(&in.From, &out.From, 0); err != nil {
 		return err
 	}
-	switch in.From.Kind {
```

Comment:
I am going over the generated conversions and they seem to be covering all of the manual so I'll probably remove these.